app/assets/stylesheets/core/_core.scss

```diff
@@ -15,7 +15,7 @@ body {
 body {
   background-color: $lighter-blue;
   color: $dark-grey;
-  font: 400 0.8em/1.2em $fonts;
+  font: 400 14px/1.2 $fonts;
```

Comment:
`14px/1.2` should be written with a single space on each side of the operator: `14px / 1.2`